showcase/src/assets/examples/table/with-custom-styling.scss

```diff
@@ -0,0 +1,19 @@
+.even {
+  background-color: #d1e1ff;
+}
+
+.danger {
+  color: #7c002c;
+}
+
+.warning {
+  color: #ff9800;
+}
+
+.success {
+  color: #3c9f40;
+}
+
+.bold {
+  font-weight: bold;
```

Comment:
This style is only applied to the Header of the column but not to all its cells 😞 

![table-column-styles-issue](https://user-images.githubusercontent.com/5698318/48561094-8a4e1180-e8ef-11e8-84d2-355f7baf98f8.PNG)